tests/utils.py

```diff
@@ -56,3 +57,7 @@ def enable(self):
 
     def disable(self):
         registry._methods = self.old_methods
+
+
+def totp_str(key):
+    return str(totp(key)).zfill(6)
```

Comment:
Minor nitpick: we should call `totp_digits()`, otherwise we'll get unnecessary test failures if `TWO_FACTOR_TOTP_DIGITS` ever changes.